### PR TITLE
double trouble by Yevhen Sahach

### PR DESCRIPTION
**Pull request by eugeneSahach**

### CODE REVIEW

File: src/doubleTrouble.js [hunk 1]
```diff
@@ -24,7 +24,10 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(numbers, target) {
-  // write code here
+  return numbers
+    .filter((elem, i) => numbers[i - 1] + numbers[i] !== target
+      || (numbers[i - 2] + numbers[i - 1] === target
+      && numbers[i] !== numbers[i - 1]));
```
Comment (irenhh) [inline src/doubleTrouble.js]: why did you add so many conditions?

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/doubleTrouble.js (hunk 1 above)
Comment (yevhen-sahach) [reply src/doubleTrouble.js]: @irenhh there is only three conditions, and I can't pass one test without second and third conditions

**Review by irenhh — APPROVED** [review]
